### PR TITLE
Implement aliasQuery

### PR DESCRIPTION
**Pull request by leizor**
This PR adds an implementation for the `aliasQuery` function.

Graphite doc: https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.aliasQuery
Graphite-web implementation: https://github.com/graphite-project/graphite-web/blob/02bc0c836269e07b38175fbfe3b5689afc87b4fa/webapp/graphite/render/functions.py#L2587-L2610

### CODE REVIEW

File: expr/functions/aliasQuery/function.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+		if err != nil {
+			return nil, merry.WithHTTPCode(err, 400)
+		}
+
+		n := fmt.Sprintf(newName, v)
+
+		var r *types.MetricData
+		if series.Name == n {
+			r = series.CopyLinkTags()
+			r.Tags["name"] = r.Name
```
Comment (npazosmendez) [inline expr/functions/aliasQuery/function.go:78]: Is this needed? Wouldn't the tag be the same?

Comment (leizor) [reply expr/functions/aliasQuery/function.go:78]: I think it's necessary, although tbh I don't fully understand why. I'm trying to copy what was done in https://github.com/grafana/carbonapi/pull/76

File: expr/functions/aliasQuery/function.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+		fetchTargets[i] = expr
+	}
+	targetValues, err := f.GetEvaluator().Fetch(ctx, fetchTargets, from, until, values)
+
+	results := make([]*types.MetricData, len(seriesList))
+
+	for i, series := range seriesList {
+		v, err := f.getLastValueOfSeries(ctx, fetchTargets[i], from, until, targetValues)
+		if err != nil {
+			return nil, merry.WithHTTPCode(err, 400)
```
[the remark this answers is not in the document]
Comment (leizor) [reply expr/functions/aliasQuery/function.go:70]: Yea I like that better--I'll make the change!

File: expr/functions/aliasQuery/function.go (hunk 1 above)
Comment (npazosmendez) [reply expr/functions/aliasQuery/function.go:76]: Mm I see. I think it's meant to fix the input of it didn't have the `"name"` tag. :+1:

File: expr/functions/aliasQuery/function.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+		}
+
+		results[i] = r
+	}
+
+	return results, nil
+}
+
+func (f *aliasQuery) getLastValueOfSeries(ctx context.Context, e parser.Expr, from, until int64, targetValues map[parser.MetricRequest][]*types.MetricData) (float64, error) {
+	res, err := f.GetEvaluator().Eval(ctx, e, from, until, targetValues)
```
Comment (npazosmendez) [inline expr/functions/aliasQuery/function.go:88]: nitpicking: Would `GetSeriesArg` work here?

```suggestion
	res, err := helper.GetSeriesArg(ctx, e, from, until, targetValues)
```

Comment (leizor) [reply expr/functions/aliasQuery/function.go:88]: Good call, and it has that extra check in case something went horribly wrong building the expression.

**Pull request merged**